src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1185,7 +1171,7 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
 
     this.currencySymbol = getCurrencySymbol(this.campaign.currencyCode, 'narrow', 'en-GB');
 
-    if (environment.psps.stripe.enabled && this.campaign.charity.stripeAccountId) {
```

Comment:
It would still have a code change to set `enabled` to false and a new deployment wouldn't it? So I can put this back how it was if you'd like but I'm not sure to be honest it saves us much time in the event a stripe outage. And having it adds a bit of complexity when we're reading the code.

Reply:
@bdsl I think that having flags for this kind of emergency change in big campaigns is still worth it to avoid having to do anything more complicated while under time pressure.

Reply:
OK, I've put the flag back in bdb6cb05